playbooks/robusta_playbooks/event_enrichments.py

```diff
@@ -0,0 +1,128 @@
+from robusta.api import *
+
+
+class EventEnricherParams(ActionParams):
+    max_events: int = 8
+    included_types: List[str] = ["Warning", "Normal"]
+
+
+class ExtendedEventEnricherParams(EventEnricherParams):
+    """
+    :var dependent_pod_mode: when True, instead of fetching events for the deployment itself, fetch events for pods in the deployment.
+    """
+
+    dependent_pod_mode: bool = False
+
+
+@action
+def event_report(event: EventChangeEvent):
+    """
+    Create finding based on the kubernetes event
+    """
+    k8s_obj = event.obj.involvedObject
+
+    # creating the finding before the rate limiter, to use the service key for rate limiting
+    finding = Finding(
+        title=f"{event.obj.reason} {event.obj.type} for {k8s_obj.kind} {k8s_obj.namespace}/{k8s_obj.name}",
+        description=event.obj.message,
+        source=FindingSource.KUBERNETES_API_SERVER,
+        severity=FindingSeverity.INFO
+        if event.obj.type == "Normal"
+        else FindingSeverity.DEBUG,
+        finding_type=FindingType.ISSUE,
+        aggregation_key=f"Kubernetes {event.obj.type} Event",
+        subject=FindingSubject(
+            k8s_obj.name,
+            FindingSubjectType.from_kind(k8s_obj.kind),
+            k8s_obj.namespace,
+            KubeObjFindingSubject.get_node_name(k8s_obj),
+        ),
+    )
+    event.add_finding(finding)
+
+
+@action
+def event_resource_events(event: EventChangeEvent, params: EventEnricherParams):
+    """
+    Given a Kubernetes event, gather all other events on the same resource in the near past
+    """
+    if not event.get_event():
+        logging.error(
+            f"cannot run event_resource_events on alert with no events object: {event}"
+        )
+        return
+    obj = event.obj.involvedObject
+    events_table = get_resource_events_table(
+        f"*Related Events*",
+        obj.kind,
+        obj.name,
+        obj.namespace,
+        included_types=params.included_types,
+        max_events=params.max_events,
```

Comment:
Before this change we would bring all events.
This playbook only sends it to the UI by default.
Why are we limiting it?

Reply:
Yeah, we can undo it. I think you're right

Reply:
fixed